fluent-plugin-prometheus.gemspec

```diff
@@ -15,6 +15,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "fluentd", ">= 1.9.1", "< 2"
   spec.add_dependency "prometheus-client", ">= 2.1.0"
+  spec.add_dependency "zlib", ">=3.0.0"
```

Comment:
Sorry, one more thing.
This dependency isn't needed because it's a default gem (bundled in Ruby and cannot remove), and this PR doesn't use specific features in zlib >= 3.0.0.
In fact fluentd's bundled plugins also use it without such dependency.

Reply:
OK 
i will remove it
anything else?

Reply:
No, I'm going to merge this after it's removed.

Strictly speaking it would be better to check `Accept-Encoding` then response uncompressed data if it doesn't support gzip. However, since behaviour of client is known and controllable on this case as you mentioned in https://github.com/fluent/fluent-plugin-prometheus/pull/223#discussion_r1695554372, there is no need to go that far.